VERSION

```diff
@@ -1 +1 @@
-0.3.24
+1.0.0
```

Comment:
Yeah, it's not a giant leap functionality-wise, but it does change the output significantly (field function support). Also dropping out 1.8.7 support seemed like a good reason too.